title_case/title_case.rb

```diff
@@ -0,0 +1,9 @@
+def title_case(string, except = '')
+  except_list = except.downcase.split(' ')
+  w = string
```

Comment:
1. probably we don't need `w = ` variable since it is not used anywhere.
2. we probably could omit the `downcase` and the logic with checking the `index` and use `capitalize` (that will capitalize the first letter and downcase all the others.

so the final solution could look like: 

```ruby
def title_case(string, except = '')
  except_list = except.downcase.split(' ')
  string.capitalize.split.map { |w| except_list.include?(w) ? w : w.capitalize }.join(' ')
end
```

Reply:
We can't omit `downcase` because excepted words can came uppercased.
```
     expected: "The Wind in the Willows"
     got: "The Wind In The Willows"
```

If we omit index checking, then words in `except` will be downcased even in first place:

``` 
     expected: "A Clash of Kings"
     got: "a Clash of Kings"
```